src/main/scala/devices/debug/Debug.scala

```diff
@@ -722,6 +721,263 @@ class TLDebugModuleInner(device: Device, getNComponents: () => Int, beatBytes: I
       }
     }
 
+    //--------------------------------------------------------------
+    // System Bus Access
+    //--------------------------------------------------------------
+    // --- SBCS Registers ---
+
+    sb2tlOpt match {                   
+      case Some(sb2tl) => {
+       
+        /* Flags for setting error flag registers, needed because the number of SBDATA and
+         * SBADDRESS registers depends on system bus widths
+         */
+        val anyAddressWrEn = Wire(init = false.B)
+        val anyDataRdEn    = Wire(init = false.B)
+        val anyDataWrEn    = Wire(init = false.B)
+
+        // --- SBCS Status Register ---
+        val SBCSFieldsReg      = Reg(new SBCSFields())
```

Comment:
do these registers show up with reasonable names in the verilog (questionable because they are inside a code block). If not, we may want to `suggestName` on some of the key signals, or return them from this code block to get assigned some names.

Reply:
This might also be a nice use case for the `DescribedReg` utility, if they aren't getting nice names.